### ISSUE TITLE
Improve Logging in Data Science Pipelines pods

### ISSUE THREAD

**Issue by rimolive** [opendatahub-io/data-science-pipelines-tekton#148]
Every time we need to enable debug logging in DSP components (API Server, persistence agent, scheduled workflow), it is a pain to make it work. We need to enable logging through manifest parameters and inject through env var in all pods to enable debugging.

**Comment by DharmitD** [comment]
The initial scope of the task was around just making manifest changes to our midstream repo. On further investigation, we figured that we would need to make changes to the code base as well (main.go for ever component), irrespective on whether we choose to use logrus or glog/klog packages for setting logging levels.

For example, Persistence Agent doesn't have the `glog` package imported: https://github.com/opendatahub-io/data-science-pipelines/blob/master/backend/src/agent/persistence/main.go#L26
We would need to import the package, and also add some code the way we've added i[n this PR](https://github.com/opendatahub-io/data-science-pipelines/pull/160/commits/044285818008cc45ae6f4348f90490d3b8e5751a#diff-8f4e4a5ceffa55ba53d8ff62f75350c998b40e8289f3b097569e5115740398caR99), to make sure we have a `log-level` cli parameter, through which we could pass a value and set the variable for the desired log level.

Similarly, we would need to update the code base for APIServer and Scheduled Workflow components as well. Hence, this change needs to go into the upstream kfp-tekton repo first.
